mmdet3d/models/voxel_encoders/voxel_encoder.py

```diff
@@ -630,11 +630,14 @@ def forward(self, features: Tensor, coors: Tensor, *args,
         for vfe in self.vfe_layers:
             features = vfe(features)
             point_feats.append(features)
-        voxel_feats, voxel_coors = self.vfe_scatter(features, coors)
+        voxel_feats, voxel_coors, point2voxel_maps = self.vfe_scatter(
+            features, coors)
 
         if self.compression_layers is not None:
             voxel_feats = self.compression_layers(voxel_feats)
 
-        if self.return_point_feats:
-            return voxel_feats, voxel_coors, point_feats
-        return voxel_feats, voxel_coors
+        voxel_dict['voxel_feats'] = voxel_feats
+        voxel_dict['voxel_coors'] = voxel_coors
+        voxel_dict['point2voxel_maps'] = point2voxel_maps
+        voxel_dict['point_feats'] = point_feats
```

Comment:
It would be confusing, if we put `point_feats` into `voxel_dict`.

Reply:
`point_feats` should be put in `voxel_dict` for potential `PointRefineModule`.

Reply:
From my perspective, `point_feats` should not be added into `voxel_dict`, or maybe we can use `feat_dict`